tensorflow/lite/micro/kernels/unidirectional_sequence_lstm_test.cc

```diff
@@ -547,218 +2417,409 @@ QuantizationParams SetQuantizationParams(float f_min, float f_max) {
   return qparam;
 }
 
-void TestUnidirectionalSequenceLstmInteger(
+void CopyInputOrExpectedOutput(const float* input, float* dest,
+                               bool batch_major, int sequence_length,
+                               int n_batch, int n_input_output) {
+  if (batch_major) {
+    memcpy(dest, input,
+           n_batch * sequence_length * n_input_output * sizeof(float));
+  } else {
+    for (int s = 0; s < sequence_length; ++s) {
+      for (int b = 0; b < n_batch; ++b) {
+        const float* batch_start =
+            &input[b * sequence_length * n_input_output + s * n_input_output];
+        memcpy(&dest[s * n_batch * n_input_output + b * n_input_output],
+               batch_start, n_input_output * sizeof(float));
+      }
+    }
+  }
+}
+
+void TestUnidirectionalSequenceLstmHybrid(
     int n_batch, int n_input, int n_cell, int n_output, int sequence_length,
     bool time_major, bool use_cifg, bool use_peephole,
     bool use_projection_weights, bool use_projection_bias, bool use_layer_norm,
-    bool use_8x8_8_implementation, const float ranges[][2],
-    float intermediate_scale[5][2], int intermediate_zp[5][2],
-    TfLiteAffineQuantization intermediate_qparam[5],
+    float cell_clip, float proj_clip,
 
-    const float* input, int8_t* input_quantized,
+    const float* input_original, float* input,
```

Comment:
not strictly required, but I personally favor one def per line.   https://yaqs.corp.google.com/eng/q/5304356331585536.

Reply:
I agree with the one def per line rule for variable declarations, but this rule may not apply to function parameters. I tried to separate them into individual lines but clang-format would always group them back into one line.